src/commands/flow/spec/flow.spec.ts

```diff
@@ -0,0 +1,187 @@
+import { AuthConfig, Target } from '@sasjs/utils'
+import * as internalModule from '../internal/'
+import SASjs, { PollOptions } from '@sasjs/adapter/node'
+import { execute } from '../execute'
+
+describe('sasjs flow', () => {
+  describe('execute', () => {
+    it('should execute flow in order - basic', async () => {
+      const flows = {
+        flow1: {
+          jobs: [{ location: 'somejob' }],
+          predecessors: []
+        },
+        flow2: {
+          jobs: [{ location: 'somejob' }],
+          predecessors: ['flow1']
+        }
+      }
+
+      setupMocks(flows)
+
+      await execute(
```

Comment:
you can put it into a variable to avoid code duplication

Reply:
wrapped to  `executeFlowWrapper`